internal/binutils/addr2liner.go

```diff
@@ -70,7 +70,11 @@ func (a *addr2LinerJob) write(s string) error {
 }
 
 func (a *addr2LinerJob) readLine() (string, error) {
-	return a.out.ReadString('\n')
+	s, err := a.out.ReadString('\n')
```

Comment:
What is the motivation for moving this from readString() to readLine()?

Reply:
This it's more obvious that every `ReadString("\n")` from the buffer has been properly trimmed. Otherwise it's easy to lose track of the location. But, I don't think it's strictly necessary.